src/plugins/discourse/mirror.test.js

```diff
@@ -89,19 +138,48 @@ class MockFetcher implements Discourse {
   }
 
   _topic(id: TopicId): Topic {
-    return {
-      ...this._topicView(id),
-      bumpedMs: 1000,
-    };
+    const view = this._topicView(id);
+    const posts = (this._topicToPostIds.get(id) || []).map((pid) =>
+      this._post(pid)
+    );
+    let bumpedMs = view.timestampMs;
+
+    // Category definition topics don't support bumping, as they are never
+    // in any API responses that contain this field.
+    if (!this._categoryDefinitionTopics.has(id)) {
+      bumpedMs = posts.reduce(
```

Comment:
Since we already depend on d3-array, I would personally go for:

```js
import {max} from "d3-array";

bumpedMs = max(posts, (p) => p.timestampMs)
```

Implementation is [quite reasonable](https://github.com/d3/d3-array/blob/master/src/max.js) at 20 LOC.

Reply:
Using max now to do the max + pluck.
Checking for null is now done in the lines above.